CHANGELOG.md

```diff
@@ -414,7 +414,7 @@ the VCS metrics to include `vcs.repository.name` as a recommended attribute.
 ### 🧰 Bug fixes 🧰
 
 - `service`: Merge `resource` experimental and stable groups for service and telemetry.sdk (#1423)
-  Discovered when fixing [weaver#306](https://github.com/open-telemetry/weaver/issues/306#issue-2458430277)
```

Comment:
This was a valid link fragment.
Why was it removed?

Reply:
because lychee can't validate it, and I didn't think that linking to the PR description was any better than linking to the PR